src/domain/home/form/HomePreliminaryFormUtils.ts

```diff
@@ -0,0 +1,37 @@
+import { get } from 'lodash';
+
+import { RegistrationFormValues } from '../../registration/types/RegistrationTypes';
+import { DEFAULT_DATE_FORMAT } from '../../../common/time/TimeConstants';
+import { newMoment } from '../../../common/time/utils';
+import { HomeFormValues } from './types/FormTypes';
+
+export const convertFormValues = (
```

Comment:
function name should be getInitialFormValues

Reply:
It feels stupid to send initialFormValues to a getInitialFormValues function..

Reply:
😆